aws-observability/templates/sumologic_observability.master.template.yaml

```diff
@@ -53,8 +53,8 @@ Metadata:
         Parameters:
```

Comment:
`7. Sumo Logic AWS Lambda CloudWatch Logs Source` -> `7. Sumo Logic CloudWatch Logs Source` as discussed with PM we will rename this section